cmd/fissh/main.go

```diff
@@ -86,8 +98,13 @@ func teaHandler(s ssh.Session) (tea.Model, []tea.ProgramOption) {
 	appStyle := renderer.NewStyle().AlignHorizontal(lipgloss.Center)
 
 	ip := s.RemoteAddr().(*net.TCPAddr).IP
-	timezone := timezone.LookupTimezone(ip.String())
-	loc, err := time.LoadLocation(timezone)
+
+	foundTimeZone, err := extractTimezoneFromEnv(s.Environ())
```

Comment:
is there a reason to rename this to `foundTimeZone` from just `timezone`?

Reply:
I had renamed it to not clobber the timezone package import. Previously, shadowing it with the local assignment was fine because we only needed the library at the only and only assignment of that var but now we needed it after the first potential assignment. I'm happy to change the name to smth else tho if y'all'd prefer.